src/main/java/ch/njol/skript/bukkitutil/BukkitUtils.java

```diff
@@ -18,4 +19,20 @@ public static boolean registryExists(String registry) {
 		return Skript.classExists("org.bukkit.Registry") && Skript.fieldExists(Registry.class, registry);
 	}
 
+	/**
+	 * Get an instance of the {@link PotionEffectType} {@link Registry}
+	 * <p>Paper/Bukkit have 2 different names for the same registry.</p>
+	 *
+	 * @return PotionEffectType Registry
+	 */
+	@SuppressWarnings("NullableProblems")
+	public static Registry<PotionEffectType> getPotionEffectTypeRegistry() {
+		if (registryExists("MOB_EFFECT")) { // Paper (1.21.4)
+			return Registry.MOB_EFFECT;
+		} else if (registryExists("EFFECT")) { // Bukkit (1.21.x)
+			return Registry.EFFECT;
+		}
+		return null;
```

Comment:
It might be better to throw an exception here (maybe like IllegalState or UnsupportedOperation)

Reply:
I use the nullness in the BukkitClasses class:
<img width="904" alt="Screenshot 2025-01-03 at 10 51 07 AM" src="https://github.com/user-attachments/assets/a4a4604e-7592-49d7-a7fe-98bde013fde8" />

Reply:
I would annotate the method as Nullable then